src/test/ui/issues/issue-59494.stderr

```diff
@@ -7,8 +7,6 @@ LL |     let t8 = t8n(t7, t7p(f, g));
    |              required by a bound introduced by this call
    |
    = help: the trait `Fn<(_,)>` is not implemented for `impl Fn(((_, _), _))`
-   = note: expected a closure with arguments `(((_, _), _),)`
-              found a closure with arguments `(_,)`
```

Comment:
Interesting regression 🤔

Reply:
It's not a regression, per se. The original note was nonsense, because the reason that this `impl Fn(((_, _), _))` does not implement `Fn<(_,)>` is unrelated to arg mismatch and has to do with some cyclical closure stuff. This was just revealed because now `(((_, _), _),)` and `(_,)` are "same type modulo infer".

Reply:
Ah! I see. Neat.